pkg/builder/controller.go

```diff
@@ -167,6 +167,10 @@ func (blder *Builder) Build(r reconcile.Reconciler) (controller.Controller, erro
 	if blder.forInput.err != nil {
 		return nil, blder.forInput.err
 	}
+	// Checking the reconcile type exist or not
+	if blder.forInput.object == nil {
+		return nil, fmt.Errorf("must provide an object for reconciliation")
```

Comment:
please use `errors.New` for error messages without a format string

Reply:
@alvaroaleman In the same func, fmt.Errorf is used for other errors which do not have a format string. IMHO this needs to be fixed horizontally for all such error messages.